app.py

```
@@ -1929,11 +2353,15 @@ async def api_who_targets(req: WHOTargetsRequest) -> WHOTargetsResponse:
     try:
         import sys as _sys
 
-        # Resolve via the 'app' package so tests can patch app.build_nutrition_targets
-        _app_pkg = _sys.modules.get("app")
-        _getattr = getattr(_app_pkg, "getattr", getattr)
-        _build_targets = _getattr(_app_pkg, "build_nutrition_targets", None)
-        if _build_targets is None:
+        # Resolve through multiple module candidates so monkeypatched attributes are respected
+        _candidates = [
```

Flake8 (7.3.0)

[error] local variable '_candidates' is assigned to but never used

(F841)